site/components/wasm-whitepaper-slice/index.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react';
+
+import styles from './style.css';
+import titleImg from './images/title.png';
+import whitepaperImg from './images/whitepaper.png';
+
+const WasmWhitepaperSlice = () => {
+  return (
+    <div className={`${styles.WasmWhitepaperSliceWrapper}`} id="genz">
+      <div className={styles.WasmWhitepaperSliceContent}>
+        <div className={styles.WasmWhitepaperSlice}>
+          <div className={styles.WasmWhitepaperSliceText}>
+            <img
+              className={styles.WasmWhitepaperSliceTitleImage}
+              src={titleImg}
+              alt="text reading: We Love Tech"
+            />
+            <div className={styles.WasmWhitepaperSliceTextContent}>
+              {/* <div className={styles.WasmWhitepaperSliceCTA}>
+                <h5>DOWNLOAD THE WHITE PAPER</h5>
+              </div> */}
+              <div className={styles.WasmWhitepaperSliceBodyheader}>
+                <h3>
+                  Multi-cloud platforms are here: How WebAssembly, NATS and wasmCloud can move us
+                  beyond the cloud
+                </h3>
+              </div>
+              <div className={styles.WasmWhitepaperSliceBodySubHeader}>
+                <h4>
+                  With contributions from Liam Randall, Cosmonic’s CEO and Derek Collison, CEO of
+                  Synadia and creator of <a href="http://nats.io/">NATS.io</a>, our latest white
```

Comment:
check with marketing if they want this in a new tab.

Reply:
Good shout. Same tab so all good!